package.json

```
@@ -32,11 +31,11 @@
     "request": "^2.87.0",
     "rimraf": "2",
     "semver": "~5.3.0",
-    "tar": "^2.0.0",
+    "tar": "4",
```

Should this be?:

```suggestion
    "tar": "^4.4.8",
```

Otherwise users without `npm audit` may still get the unsecure version.

---

this is the only part left in 6bc9bf1 so I'm removing that commit entirely and it'll end up with `^4.4.8`